terraform/deployments/tfc-configuration/variables.tf

```diff
@@ -0,0 +1,289 @@
+  default     = []
+}
+
+variable "terraform_version" {
+  type        = string
+  description = "Version of Terraform to use for this Workspace."
+  default     = null
+}
+
+variable "trigger_prefixes" {
+  type        = list(string)
+  description = "List of paths relative to the root of the VCS repo to filter on when `file_triggers_enabled` is `true`."
+  default     = null
+}
+
+variable "trigger_patterns" {
+  type        = list(string)
+  description = "List of glob patterns that describe the files monitored for changes to trigger Runs in Workspace. Mutually exclusive with `trigger_prefixes`. Only available with TFC."
+  default     = null
+}
+
+variable "working_directory" {
+  type        = string
+  description = "The relative path that Terraform will execute within. Defaults to the root of the repo."
+  default     = null
+}
+
+variable "vcs_repo" {
+  type        = map(string)
+  description = "Map of settings to connect Workspace to VCS repository."
+  default     = {}
+}
+
+variable "tags_regex" {
+  type        = string
+  description = "A regular expression used to trigger a Run in Workspace for matching Git tags. This option conflicts with `trigger_patterns` and `trigger_prefixes`. Should only set this value if the former is not being used."
+  default     = null
+}
+
+variable "force_delete" {
+  type        = bool
+  description = "Boolean to allow deletion of the Workspace if there is a Terraform state that contains resources."
+  default     = null
+}
+
+variable "project_name" {
+  type        = string
+  description = "Name of existing Project to place Workspace in."
+  default     = null
+}
+
+#------------------------------------------------------------------------------
+# Workspace Variables
+#------------------------------------------------------------------------------
+variable "tfvars" {
+  type        = any
+  description = "Map of Terraform variables to add to Workspace."
+  default     = {}
+}
+
+variable "tfvars_sensitive" {
+  type        = any
+  description = "Map of sensitive Terraform variables to add to Workspace."
+  default     = {}
+}
+
+variable "tfvars_ignore_changes" {
+  type        = any
+  description = "Map of Terraform variables to add to Workspace whereby changes made outside of Terraform will be ignored."
+  default     = {}
+}
+
+variable "envvars" {
+  type        = map(string)
+  description = "Map of Environment variables to add to Workspace."
+  default     = {}
+}
+
+variable "envvars_sensitive" {
+  type        = map(string)
+  description = "Map of sensitive Environment variables to add to Workspace."
+  default     = {}
+}
+
+variable "envvars_ignore_changes" {
+  type        = map(string)
+  description = "Map of sensitive Environment variables to add to Workspace whereby changes made outside of Terraform will be ignored."
+  default     = {}
+}
+
+# #------------------------------------------------------------------------------
```

Comment:
Seems reasonable to keep these commented-out parts in for illustration 👍

Reply:
Yeah i figured it was easier to see them there rather than having to go to the module to look for them